src/mappings/Cache.ts

```diff
@@ -31,22 +31,26 @@ export async function cachedRewardDestination(accountAddress: string, event: Sub
         // looks like accountAddress not related to events so just try to query payee directly
         if (allAccountsInBlock.length === 0) {
             rewardDestinationByAddress[blockId] = {}
+            // @ts-ignore
```

Comment:
Why do we move this lines to ignore?

Reply:
I have updated types to remove all `// @ts-ignore`
- Replaced RewardDestination type to PalletStakingRewardDestination to resolve:
`It has a typescript problem - Type 'PalletStakingRewardDestination' is missing the following properties from type 'RewardDestination': #private, toPrimitivets(2739)`
- Updated to polkadot/api v8 to resolve:
```
TS2345: Argument of type 'import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/types/calls").CallBase<import
 ›   ("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types-codec/types/helpers").AnyTuple, 
 ›   import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/interfaces/metadata/types").FunctionMeta...' is not 
 ›   assignable to parameter of type 'import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/types/calls").CallBase<impor
 ›   t("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/node_modules/@polkadot/types-codec/types/helpers").AnyTuple, 
 ›   import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/interfaces/...'.
 ›     Type 'import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types-codec/types/helpers").AnyTuple' is not assignable to 
 ›   type 'import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/node_modules/@polkadot/types-codec/types/helpers").AnyT
 ›   uple'.
 ›       Property 'toPrimitive' is missing in type 
 ›   'import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types-codec/types/codec").Codec' but required in type 
 ›   'import("/Users/stepanlavrentev/subquery-nova/node_modules/@polkadot/types/node_modules/@polkadot/types-codec/types/codec").Codec'.
```
 - add `as unknown as Balance` to resolve:
 https://stackoverflow.com/a/69399373/19548081
```
Conversion of type 'Codec' to type 'Balance' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```